internal/events/delivery_test.go

```diff
@@ -205,7 +205,7 @@ func failureEventHandler(t *testing.T, deliveryChan chan<- *model.StateChangeEve
 }
 
 func awaitEvents(t *testing.T, deliveryChan <-chan *model.StateChangeEventPayload, expectedID string, eventsCount int) {
-	timeout := time.After(25 * time.Second)
+	timeout := time.After(50 * time.Second)
```

Comment:
Is this change related to the scope of the PR?

Reply:
Ok, I now have context https://community.mattermost.com/private-core/pl/8gycrgzoij8hdm6ogiines7zxe